polkadot/runtime/rococo/src/lib.rs

```diff
@@ -1028,13 +1028,18 @@ impl coretime::Config for Runtime {
 
 parameter_types! {
 	pub const OnDemandTrafficDefaultValue: FixedU128 = FixedU128::from_u32(1);
+	// Keep 2 timeslices worth of revenue information.
+	pub const MaxHistoricalRevenue: BlockNumber = 2 * 80;
```

Comment:
We should be able to retrieve this value from [here](https://github.com/paritytech/polkadot-sdk/blob/99234440f0f8b24f7e4d1d3a0102a9b19a408dd3/cumulus/parachains/runtimes/coretime/coretime-rococo/src/coretime.rs#L227)

Reply:
We could, but I'm asking myself if it is right to create a dependency of a relay chain runtime to a parachain pallet. A common constant could work, and here I should ask which side should define that constant. IIUC, the timeslice length cannot be called a property of either parachain or relay chain; instead, it's a property of the whole system formed by the relay chain and the coretime parachain. Being put like that, it seems that putting it into `rococo-runtime-constants` (and separately into `westend-runtime-constants`) is the most natural move. `coretime-rococo` and `coretime-westend` runtimes naturally depend on the corresponding relay chain constants anyway. Let me know if I'm looking at the wrong angle.

Reply:
Alternatively we could hardcode them both as we do now, but ensure that they are correct with a straightforward emulated test, since in emulated tests we have both runtimes as dependencies. That means if somebody changes one the test will fail. It's a similar strategy I'm planning to use for hardcoded weights between runtimes